generators/java/model/CHANGELOG.md

```diff
@@ -10,19 +10,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Feature: The generator now supports BigInteger types.
 - Chore: Bump intermediate representation to v46
 
-## [0.8.2] - 2024-06-04
```

Comment:
we couldn't find the sdk changelog for the previous exception release because i put it here in `model` 🤦